src/network/networkd-manager.c

```diff
@@ -607,6 +607,7 @@ int manager_new(Manager **ret, bool test_mode) {
                 .duid_product_uuid.type = DUID_TYPE_UUID,
                 .dhcp_server_persist_leases = true,
                 .ip_forwarding = { -1, -1, },
+                .sysctl_shadow = hashmap_new(&string_hash_ops),
```

Comment:
actually, please drop this. please initialize this lazily. but plesae pass a pointer to a pointer as userdata to the inotify callback so that you can lazily initialize the thing after all.

also, you used different hash ops here